src/backend/utils/word-counter.js

```diff
@@ -1,5 +1,8 @@
+/**
+ * https://www.npmjs.com/package/wordcount
+ */
 const wordcount = require('wordcount');
 
-exports.wordCounter = async function wordCounter(blogText) {
-  return Promise.resolve(wordcount(blogText));
+exports.wordCounter = function wordCounter(blogText) {
```

Comment:
This file isn't doing anything.  We're pulling in the `wordcount` module, then creating a function called `wordCounter` that calls `wordcount`.  Let's just `git rm` this file, and change the caller(s) to use this module directly.